about.html

```diff
@@ -0,0 +1,115 @@
+        <li><a href="#"><i class="fa fa-facebook-f" aria-hidden="true"></i></a></li>
+        <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+      <div class="nav-menu">
+        <i id="close-menu" class="fas fa-times fa-3x" aria-hidden="true"></i>
+        <a href="./index.html"><img class="nav-image" src="./media/institute_logo.png" alt="Logo image"></a>
+        <ul class="about-bottom-nav">
+          <li>
+            <a href="./about.html">About</a>
+          </li>
+          <li>
+            <a href="./index.html#program">Program</a>
+          </li>
+          <li>
+            <a href="./index.html#features">Featured Films</a>
+          </li>
+          <li>
+            <a href="#">Sign up!</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </header>
+
+  <main>
+    <i id="open-menu" class="fa fa-bars fa-3x" aria-hidden="true"></i>
+    <section id="about-main">
+      <div class="about-wrapper">
+        <h2>Hello film world!</h2>
+        <h1>Sundance Film Festival 2023</h1>
+        <p class="about-festival">The Sundance Film Festival is the ultimate gathering of original storytellers and audiences seeking new voices and fresh perspectives. Our annual program includes dramatic and documentary features and short films;
+          series and episodic content; and New Frontier, showcasing emerging media in the form of multimedia installations, performances, and films.</p>
+        <p class="contact-text">Please contact us per Email for any further questions</p>
+        <p class="contact-email">festival2023@sundace.com</p>
+      </div>
+    </section>
+
+    <section id="about-institute">
+      <div class="institute-wrapper">
+        <h2 class="h2dark">The Sundance Institute</h2>
+        <div class="separator"></div>
+        <div class="institute-text">
+
+          <p><q>Sundance deepens the resolve of artists, bringing them together in a creative community so that they know they are not alone.</q> — Robert Redford, President and Founder</p>
+          <p>The Sundance Institute’s artist programs provide dynamic support at every step of the creative journey for individuals with distinct voices in film, theatre, film composing, episodic storytelling, and emerging media. Each program
+            consists of labs, grants, intensives, sessions, and ongoing resources for artists to nurture their projects and sustain their careers. Through year-round activities, we provide the space for artists to create and share their stories with
+            the world. Since our inception, the Institute’s programs have supported more than 11,000 artists and fostered a community for independent storytellers to learn, grow, connect, and give back.</p>
+          <img src="./media/institute_logo.png" alt="Film image">
+        </div>
+      </div>
```

Comment:
- [ ] Please use the same design as defined in the original design. You can clearly see the difference from the original please work on it and make them the same  cause you need to stick the original design as much as possible :smile: 

#### Original 
![Screenshot from 2022-07-28 02-06-52](https://user-images.githubusercontent.com/6615394/181404861-3c6a082f-304c-49bd-bd30-0b28c143ebe4.png)

#### Your design
![Screenshot from 2022-07-28 02-06-41](https://user-images.githubusercontent.com/6615394/181404870-0a6f23f7-ebc7-4d4f-a706-c8064d5ecc74.png)